src/stores/cookie-reference.md

```diff
@@ -285,5 +285,17 @@ JavaScript Library: `ga.js`
 - `_utmz`: Saves the traffic source or campaign that explains how the shopper reached your site. The cookie is created when the JavaScript library executes, and is updated every time data is sent to Google Analytics.
 - `__utmv`: Stores visitor-level custom variable data. This cookie is created when a developer uses the `_setCustomVar` method with a visitor-level custom variable. This cookie is updated every time data is sent to Google Analytics.
 
+{:.ee-only}
+## Product Recommendations Cookies
+
+The following cookies are used by Product Recommendations for Magento Commerce customers. These cookies are installed with the [DataServices module](https://devdocs.magento.com/recommendations/install-configure.html).
+
+- `mg_dnt`: Allows you to [restrict Magento data collection](https://devdocs.magento.com/recommendations/setting-cookie.html) if you have custom code to manage cookie consent on your site.
+- `user_allowed_save_cookie`: Used for Magento's [cookie restriction mode]({% link stores/compliance-cookie-restriction-mode.md %}).
+- `authentication_flag`: Tracks a shopper's sign in and sign out status.
+- `dataservices_customer_id`: Tracks a shopper's sign in and sign out actions.
+- `dataservices_cart_id`: Used with a shopper's cart actions.
```

Comment:
```suggestion
- `dataservices_cart_id`: Tracks a shopper's shopping cart.
```
let's be consistent

Reply:
I thought we were avoiding the word "Track" or "Tracking"

Reply:
@misha-kotov thoughts on Ross's question?

Reply:
yeah, agree, we should find an alternative. any leads from existing cookie docs?

Reply:
Maybe "identifies" ?